res/database/Default/database_people.xml

```diff
@@ -322,116 +322,116 @@
-			<last_name>Gebhardt</last_name>
+			<first_name>Hein</first_name>
+			<last_name>Erzhart</last_name>
 			<first_name_original>Heinz</first_name_original>
 			<last_name_original>Erhardt</last_name_original>
 			<details job="2" gender="1" birthday="1909-02-20" deathday="1979-06-05" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
-		<person id="1b8fcb9a-5961-43b8-af1b-1ed144880908" tmdb_id="0" imdb_id="nm0000154" creator="">
-			<first_name>Meal</first_name>
-			<last_name>Gibtsschon</last_name>
+		<person id="1b8fcb9a-5961-43b8-af1b-1ed144880908" tmdb_id="0" imdb_id="nm0000154" creator="" comment="previous spoof name: Meal Gibtsschon - TODO: vielleicht anderweitig verwursten">
+			<first_name>Moe</first_name>
+			<last_name>Glibsen</last_name>
 			<first_name_original>Mel</first_name_original>
 			<last_name_original>Gibson</last_name_original>
 			<details job="2" gender="1" birthday="1956-01-03" deathday="" country="" />
 		</person>
 		<person id="becc2ecd-d309-46a1-b9c2-30253915af0c" tmdb_id="0" imdb_id="nm0000027" creator="">
-			<first_name>Aleg</first_name>
+			<first_name>Alex</first_name>
 			<last_name>Ginnesse</last_name>
 			<first_name_original>Alec</first_name_original>
 			<last_name_original>Guinness</last_name_original>
 			<details job="2" gender="1" birthday="1914-04-02" deathday="2000-08-05" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="1f276c5b-2394-4a01-9ed1-d82a052dca57" tmdb_id="0" imdb_id="nm0089217" creator="">
-			<first_name>Porando</first_name>
-			<last_name>Gloom</last_name>
+			<first_name>Orlondo</first_name>
+			<last_name>Bluehm</last_name>
 			<first_name_original>Orlando</first_name_original>
 			<last_name_original>Bloom</last_name_original>
 			<details job="2" gender="1" birthday="1977-01-13" deathday="" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="5d267d5d-70a1-4b04-8bd5-bd32ffb87659" tmdb_id="0" imdb_id="nm0000418" creator="">
-			<first_name>Danny</first_name>
-			<last_name>Gloverson</last_name>
+			<first_name>Donny</first_name>
+			<last_name>Glaver</last_name>
 			<first_name_original>Danny</first_name_original>
 			<last_name_original>Glover</last_name_original>
 			<details job="2" gender="1" birthday="1946-07-22" deathday="" country="" />
 			<data popularity="20" affinity="0" fame="0" scandalizing="0" price_mod="1" power="0" humor="0" charisma="0" appearance="0" topgenre="0" />
 		</person>
 		<person id="267e028a-d087-4fff-83fd-0ade0806ab8a" tmdb_id="0" imdb_id="nm0000336" creator="">
-			<first_name>James</first_name>
-			<last_name>Gobourner</last_name>
+			<first_name>Jess</first_name>
```

Comment:
There's a "soft rule" that if a name ends with s, the spoof name should also, to avoid problems with grammar in descriptions (if it's "James' best role" then "Jess' best role" also works, but "Jim' best role" doesn't). Not that this rule is always followed, just have to keep an eye on "mistakes" that may pop up once in a while.

Reply:
Point made.